samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -163,11 +163,34 @@ Location tracking
 =================
 
 All matters concerning location tracking are handled in the :file:`src/location_tracking.c` file.
+<<<<<<< HEAD
```

Comment:
merge conflict

Reply:
Wow, not sure how that made it into a **_commit_**, my bad

Reply:
it isn't hard to do.